tests/helpers.py

```diff
@@ -25,16 +26,6 @@
 RANDOM = random.Random(b"bA\xcd\x00\xa9$\xa7\x17\x1c\x10")
 
 
-# TODO: Remove when dropping 3.9
```

Comment:
not sure why it is removed given the comment

Reply:
The comment was wrong. If you look at the condition, it checks if the version is *less than* 3.9, so the `if` branch was only being taken on 3.8. `randbytes` was added in 3.9 actually